src/pages/change-password/index.tsx

```diff
@@ -7,12 +7,18 @@ import { useRouter } from "next/router"
 import { FieldValues, SubmitHandler, useForm } from "react-hook-form"
 import { FormDataItem } from "@/interfaces/form-data-item"
 import ErrorMessage from "@/components/error/error-message"
+import TermsAndConditions from "@/components/overlays/terms-and-conditions"
+import { createPortal } from "react-dom"
+import { CheckIcon } from "@heroicons/react/24/solid"
 
 const Index = () => {
   const supabase = useSupabaseClient<Database>()
   const user = useUser()
   const router = useRouter()
   const [errorMessage, setErrorMessage] = useState("")
+  const [showTerms, setShowTerms] = useState(false)
+  const [isChecked, setIsChecked] = useState(false)
```

Comment:
I'd recommend using react-hook-form for the checked elements since the whole form uses react-hook-form
![image](https://github.com/ASVGay/the-rhapsodies/assets/54884540/090aa1d8-158f-4e99-a266-648e1e5d185f)

Reply:
I have a version that uses react-hook-forms, the only problem is that i can't change the checkbox's color due to the browser automaticaly changing it to dark. (accent-color default behaviour)
![image](https://github.com/ASVGay/the-rhapsodies/assets/4838158/d3c907e5-de1f-464b-9a6c-455b8c918ec6)

Other version i have is with some hacky-wizard-css-stuff. Only issue is that the hover styling isn't usable with that one
![image](https://github.com/ASVGay/the-rhapsodies/assets/4838158/5fbb3736-0e86-45a1-ac90-48c587ad8261)

Reply:
What hover styling are you talking about?

With the code in the picture, this is the result I get:
![msedge_x42oT8sL8u](https://github.com/ASVGay/the-rhapsodies/assets/54884540/3c8465b1-ea19-4e02-91f6-81aed1a03c2c)

Reply:
Apologies! hadn't committed that version to GIT yet. i've pushed the changes as of now.

Reply:
> I'd recommend using react-hook-form for the checked elements since the whole form uses react-hook-form ![image](https://user-images.githubusercontent.com/54884540/246158595-090aa1d8-158f-4e99-a266-648e1e5d185f.png)

I did not necessarily mean that you should create a FormDataItem for it, but literally the code in the picture so that it uses the same hooks for the form instead of creating a new separate one

Reply:
Got it, reverted previous commit and added recommended changes.